wiki/es/es-Server-Rpi.md

```diff
@@ -1,7 +1,7 @@
 ---
+category: hidden
```

Comment:
You can and should delete this file since it only needs to exist once (for the english version)

Reply:
Done. The trouble is that all files in the EN directory will have their template and .po file created. But if we leave them untranslated they won't get coverted to .md. This one probably got translated (the same with the other redirects which I'll remove).

Reply:
Ah. Ok.